yellowbrick/text/tsne.py

```diff
@@ -319,7 +350,7 @@ def finalize(self, **kwargs):
         self.ax.set_xticks([])
 
         # Add the legend outside of the figure box.
-        if self.classes_:
+        if not all(self.classes_ == np.array([self.NULL_CLASS])):
```

Comment:
This is terrible; basically, a check to see if we should add a legend or not; open to suggestions here.

Reply:
I am not sure how to handle or even advise you on this because normally without a label, the legend wouldn't be drawn even if you specify ax.legend(...)